src/server/cmd/pachctl/cmd/cmd.go

```diff
@@ -373,7 +389,8 @@ kubectl %s port-forward "$pod" %d:8080
 				if err := cmdutil.RunIO(cmdutil.IO{
 					Stdin: stdin,
 				}, "sh"); err != nil {
-					return fmt.Errorf("Could not forward dash websocket port")
+					fmt.Printf("Is the dashboard deployed? If not, deploy with \"pachctl deploy local --dashboard-only\"")
```

Comment:
Could we send this to `stderr` instead of `stdout`.

Reply:
Done